mailscanner/geoip_update.php

```diff
@@ -58,9 +61,9 @@
     ob_start();
     echo __('downfile15') . '<br>' . "\n";
 
-    $files_base_url = 'http://geolite.maxmind.com';
+    $files_base_url = 'http://download.maxmind.com';
```

Comment:
Does it also work with https? Might be worth to use it to prevent MitM and resulting attack vectors.

Reply:
Done, and it works.